sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -19,6 +22,13 @@ Azure AI Document Intelligence ([previously known as Form Recognizer][service-re
 python -m pip install azure-ai-documentintelligence
 ```
 
+This table shows the relationship between SDK versions and supported API versions of the service:
+|SDK version|Supported API version of service
+|-|-
+|1.0.0b1 | 2023-10-31-preview
+
+Older API versions are supported in `azure-ai-formrecognizer`, please check the relationship from [there](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/README.md#install-the-package). 
```

Comment:
Should we focus on linking the migration guide to encourage folks to move on to this new package?

Reply:
Added migration guide link.